taskcluster/examples-mic_vad_streaming-py39.yml

```diff
@@ -0,0 +1,13 @@
+build:
+  template_file: examples-base.tyml
+  docker_image: "python:3.9"
+  dependencies:
+    - "linux-amd64-cpu-opt"
+  system_setup:
+    >
+      apt-get -qq -y install portaudio19-dev pulseaudio
+  args:
+    tests_cmdline: "${system.homedir.linux}/DeepSpeech/examples/mic_vad_streaming/test.sh 3.9.0:"
+  metadata:
```

Comment:
@dag7dev My bad, you need to add `workerType: "${docker.dsTests}"` here

Reply:
@dag7dev You will also have to do that on `examples-mic_vad_streaming-py38.yml`

Reply:
Oh our friend at docker!
Do I need to do it for 3.8 file too?

Reply:
Yes please. I've pushed the fix in https://github.com/mozilla/DeepSpeech/pull/3408/commits/5c0734e0c7d74d702d80410b99ae5e1ce25851f3 to be able to run the PR